docs/docs/recipes/manage-users/README.md

```diff
@@ -4,8 +4,22 @@ sidebar_position: 6
 
 # 🧑‍🚀 Manage users
 
-TODO: briefly summary
+User management is the core feature for the admin staff.
+For example, you need to:
 
-- Manage users on the Admin Console page
-- Manage users by Logto Core management API
-- Manage user data on the storage
+- Search users
+- Add a user
+- View user activities
+- View user profile
+- Update user profile
+- Reset the user password
+- Delete a user
+- ……
+
+You can do it all:
+
+- on the [Admin Console page](./on-admin-console-page.md),
+- or by [Logto Core management API](./by-logto-core-management-api.md).
```

Comment:
"Management API" is enough. users is using Logto and they don't care the belonging of the API